lib/assets/stylesheets/modules/_search.scss

```diff
@@ -19,6 +12,48 @@
   }
 }
 
+.js .search__label {
+  @extend .govuk-visually-hidden;
+}
+
+.search__form {
+  position: relative;
+  width: 100%;
```

Comment:
Is the width needed here? This is on a form, which by default is a block level element - unless something else is overriding that that I can't see?

Reply:
Nope, I don't think it should be needed.